src/other/Comms.js

```diff
@@ -927,7 +927,7 @@
         function filterResults(request) {
             var changedFilter = {};
             for (var key in request) {
-                if (request[key] !== undefined && request[key + "_changed"] !== undefined) {
+                if (request[key + "_changed"] !== undefined) {
                     changedFilter[key] = {
                         value: request[key]
```

Comment:
This is the "deselect" issue, on first click you get a request like this:
{
paramA:  "007",
paramA_changed: true
}

When deselecting you would get:
{
paramA: undefined,
paramA_changed:true
}

Which was getting filtered out by the above code.